go.mod

```diff
@@ -0,0 +1,62 @@
+module github.com/vmware-tanzu/velero
+
+go 1.13
+
+require (
+	cloud.google.com/go v0.46.2 // indirect
+	github.com/Azure/azure-sdk-for-go v21.4.0+incompatible
+	github.com/Azure/go-autorest v11.1.2+incompatible
+	github.com/aws/aws-sdk-go v1.16.26
+	github.com/dgrijalva/jwt-go v3.2.0+incompatible // indirect
+	github.com/docker/spdystream v0.0.0-20170912183627-bc6354cbbc29 // indirect
+	github.com/evanphx/json-patch v0.0.0-20190203023257-5858425f7550
+	github.com/gobwas/glob v0.2.3
+	github.com/gofrs/uuid v3.2.0+incompatible
+	github.com/golang/protobuf v1.3.2
+	github.com/google/go-cmp v0.3.1 // indirect
+	github.com/googleapis/gnostic v0.1.0 // indirect
+	github.com/hashicorp/go-hclog v0.0.0-20180709165350-ff2cf002a8dd
+	github.com/hashicorp/go-plugin v1.0.1
+	github.com/joho/godotenv v1.3.0
+	github.com/pkg/errors v0.8.1
+	github.com/prometheus/client_golang v0.9.2
+	github.com/robfig/cron v1.0.1-0.20170309132418-df38d32658d8
+	github.com/sirupsen/logrus v1.4.2
+	github.com/spf13/afero v1.2.2
+	github.com/spf13/cobra v0.0.5
+	github.com/spf13/pflag v1.0.3
+	github.com/stretchr/testify v1.3.0
+	golang.org/x/net v0.0.0-20190812203447-cdfb69ac37fc
+	google.golang.org/grpc v1.21.1
+	k8s.io/api v0.0.0
+	k8s.io/apiextensions-apiserver v0.0.0
+	k8s.io/apimachinery v0.0.0
+	k8s.io/client-go v0.0.0
+	k8s.io/code-generator v0.17.0 // indirect
+	k8s.io/klog v0.3.1
+	k8s.io/kubernetes v1.15.3
+	sigs.k8s.io/controller-tools v0.2.4 // indirect
+)
+
+replace (
```

Comment:
how did you come up with these `replace` directives/why are the needed?

Reply:
These were needed to resolve errors w/ the controller-gen code generation. The way i came up w/ these was a combination of searching on the internet on how to resolve those errors and https://github.com/vmware-tanzu/velero/blob/master/Gopkg.lock#L671 replaces corresponding to the 1.15.3